config/schemas/form_response.json

```diff
@@ -97,6 +97,9 @@
         "postcode": {
           "type": "string",
           "pattern": "^[A-Za-z0-9 ]{4,}$"
+        },
+        "uprn": {
+          "type": ["string", "null"]
```

Comment:
It looks like UPRNs are always 12 digits, so do we want to add a pattern matcher here?

Reply:
Yeah good point. Jon and Graham tell me they can be under 12 digits, so I've added a `"maxLength"` of 12 there.

Reply:
And limiting it to numbers too?

Reply:
Thanks :-)